bs-react-native-next/src/apis/PanResponder.re

```diff
@@ -0,0 +1,76 @@
+type gestureState;
+
+[@bs.obj]
+external gestureState:
```

Comment:
As per https://github.com/reasonml-community/bs-react-native/blob/master/bs-react-native-next/CONTRIBUTING.md#js-objects-returned-in-callbacks, this should be a `Js.t` type.